extension_service/datastore/providers/postgres.py

```diff
@@ -184,12 +184,25 @@ async def export_data(
         flights = [models.Flight.model_validate(dict(f)) for f in await flights_task]
         return airports, amenities, flights
 
-    async def get_airport(self, id: int) -> Optional[models.Airport]:
+    async def get_airport(
+        self,
+        id: Optional[int] = None,
+        iata: Optional[str] = None,
+        name: Optional[str] = None,
+    ) -> Optional[models.Airport]:
+        id_str = "%" if id is None else str(id)
+        iata = "%" if iata is None else iata
+        name = "%" if name is None else "%" + name + "%"
+
         result = await self.__pool.fetchrow(
             """
-              SELECT id, iata, name, city, country FROM airports WHERE id=$1
+              SELECT * FROM airports WHERE CAST(id AS text) LIKE $1
```

Comment:
Could this query since it's using the like operator potentially return multiple results?

Reply:
Updated the function. Created 2 separate function and move name under search_airport.